recipes/zarp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "zarp" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0defdec6fe8dabf66d40f2f4d16dfc2619f509f11763356908af3be51ca049ab
+
+build:
+  noarch: python
+  entry_points:
+   - zarp = zarp.cli:main
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
We may need to include a `build` section and put `setuptools_git >=1.2`

Reply:
@uniqueg  Do we need snakemake as a dependency?
Is it used in zarp-cli?